snapshots/default.json

```diff
@@ -15,7 +15,7 @@
     "commit": "3022dbc"
   },
   "cmp-nvim-lsp": {
```

Comment:
could you also update `neodev`? folke has pushed some [new commits](https://github.com/folke/neodev.nvim/commits/main) :smiley:

Reply:
sure thing :)